src/components/TasksTable/index.jsx

```diff
@@ -19,6 +20,7 @@ import classNames from 'classnames';
 import DataTable from '../DataTable';
 import sort from '../../utils/sort';
 import { ASSIGNEE } from '../../utils/constants';
+import { unassigned, assigned } from '../../utils/assignment_filters';
```

Comment:
My bad, JS file name convention is to use camel case. Rename to `assignmentFilters`.